src/components/LocationPage/Experiment/LocationPageHeaderVariant.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import LocationPageHeader from '../LocationPageHeader';
+import {
+  Experiment,
+  ExperimentID,
+  Variant,
+  VariantID,
+} from 'components/Experiment';
+import { Level } from 'common/level';
+import { Metric } from 'common/metricEnum';
+import { Region } from 'common/regions';
+import type { MetricValues } from 'common/models/Projections';
+
+const LocationPageHeaderVariant: React.FC<{
+  alarmLevel: Level;
+  stats: MetricValues;
+  onMetricClick: (metric: Metric) => void;
+  onHeaderShareClick: () => void;
+  onHeaderSignupClick: () => void;
+  isMobile?: boolean;
+  region: Region;
+}> = ({
+  alarmLevel,
+  stats,
+  onMetricClick,
+  onHeaderShareClick,
+  onHeaderSignupClick,
+  isMobile,
+  region,
+}) => {
+  const headerProps = {
+    alarmLevel,
+    stats,
+    onMetricClick,
+    onHeaderShareClick,
+    onHeaderSignupClick,
+    isMobile,
+    region,
+  };
+
+  return (
+    <Experiment id={ExperimentID.MEGA_MENU_API_LINK}>
+      <Variant id={VariantID.B}>
+        <LocationPageHeader {...headerProps} />
+      </Variant>
+      <Variant id={VariantID.A}>
+        <LocationPageHeader {...headerProps} showNewButton={true} />
+      </Variant>
```

Comment:
Also, is this reversed? Shouldn't be A the original version?

Reply:
yeah i just set the new version as A for the sake of QA. will switch back before merging